src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -63,5 +67,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     response.setContentType("application/json;");
     response.getWriter().println(threadInfoList);
+
+    // get sentiment properties from text
```

Comment:
Optional: Instead of hardcoding here, you could call this in the for loop and add the values to threadSentiments (potentially on random title?). 

Otherwise, this code is currently doesn't add anything to the response.